src/platform/testing/e2e/test-server.js

```diff
@@ -28,7 +29,13 @@ if (options.unexpected && options.unexpected.length !== 0) {
 
 const app = express();
 
-const root = path.resolve(__dirname, `../../../../build/${options.buildtype}`);
+let root = path.resolve(__dirname, `../../../../build/${options.buildtype}`);
+if (!fs.existsSync(root)) {
+  // if there isn't a build directory here, then check the parent directory.
+  // This is a temporary adapation as we transition to vagov-content.
+  root = path.resolve(__dirname, `../../../../../build/${options.buildtype}`);
+}
+
```

Comment:
The test-server for E2E tests is a really weird case. It needs access to the generated files from a build in order to serve them, but those files should really only belong to content, so it's a weird relationship. I would like to see a way to more clearly define that relationship as we move forward, but it's not something that's priority at the moment.

I also wanted to leave the script files untouched between parent/child repos and not send any special args from the parent, because I felt that the child should have ownership of the script file because it's largely for testing app-code.